src/main/webapp/assets/js/feedScript.js

```diff
@@ -679,11 +892,19 @@ document.addEventListener('keydown', function(e) {
       submitModalButton.focus();
       e.preventDefault();
     }
+    if (document.activeElement == modalSubmitted) {
+      modalSubmittedText.focus();
+      e.preventDefault();
+    }
   } else { // If user is trying to go to the next element, make sure it wraps to the top.
     if (document.activeElement === submitModalButton) {
       modalCard.focus();
       e.preventDefault();
     }
+    if (document.activeElement == modalSubmittedText) {
+      modalSubmitted.focus();
+      e.preventDefault();
+    }
```

Comment:
Small nit but use `===` wherever possible! :) ([src](https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons))

Reply:
Ah good to know!